actionview/test/template/record_tag_helper_test.rb

```diff
@@ -24,94 +24,52 @@ def setup
   end
 
   def test_content_tag_for
-    expected = %(<li class="record_tag_post" id="record_tag_post_45"></li>)
-    actual = content_tag_for(:li, @post)
-    assert_dom_equal expected, actual
+    assert_raises(NoMethodError) { content_tag_for(:li, @post) }
   end
```

Comment:
Yeah, I figured that would probably be enough, but I wanted to cover all the cases. I'll slim this down.